components/astro/src/Particles.astro

```diff
@@ -1,55 +1,36 @@
 ---
-import type { Container, Engine, ISourceOptions } from "tsparticles-engine";
-
-export interface IParticlesProps {
-    id: string;
-    init: string;
```

Comment:
This is the only property that needs to be removed, following the same pattern used in React package.